andela/ah-kgl-avengers-frontend
# #167162070 Clear store when componentDidUnmount

### What does this PR do?
Clearing store when a component is unmounted
### Description of Task to be completed?
- Removing previous article contents when you want to read a new article.
### How should this be manually tested?
- Clone the repository locally with `git clone https://github.com/andela/ah-kgl-avengers-frontend.git` command.
- Check out to this branch by running `git checkout ch-clear-article-store-167162070`.
- Install dependencies by running `npm install`.
- Update the .env file according to the .env.sample file
- Start the server with `npm start` and put this url `http://localhost:8000/` in the address bar.
- Read an article
- Go back to the home page or somewhere else, then try to read a different article.
The purpose was not to show the content of the previous article
### Any background context you want to provide?
N/A
### What are the relevant pivotal tracker stories?
[#167162070](https://www.pivotaltracker.com/story/show/167162070)
### Screenshots (if appropriate)

### Questions:
N/A
